capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -58,21 +58,11 @@ export class SearchBookModal extends Component {
 
   addBookToList = (book) => {
 
-    if (this.props.type === 'club') {
-      this.setState(
-        {
-          addedBooksIDs: [book.id],
-          addedBooks: [book]
-        }
-      )
-    }
-    else {
-      this.state.addedBooksIDs.push(book.id);
-      this.state.addedBooks.push(book);
+    this.state.addedBooksIDs.push(book.id);
+    this.state.addedBooks.push(book);
```

Comment:
Same here: don't modify state directly (https://reactjs.org/docs/state-and-lifecycle.html#do-not-modify-state-directly).

Reply:
Same as above